plugin.yaml

```diff
@@ -69,3 +226,92 @@ relationships:
   cloudify.gcp.relationships.contained_in:
     derived_from: cloudify.relationships.contained_in
 
+  cloudify.gcp.relationships.agent_connected_to_security_group:
```

Comment:
Seems like the only difference between these 2 relationships types is the tag (agent, manager). Since they have the same schema but differ in values, maybe you can just use a single relationship named "instance_connected_to_security_group" and set the tag as a dynamic input value.
(Also, what is the tag used for?)